lkcrypto.c

```diff
@@ -213,6 +262,85 @@ static int decrypt_encrypt(lua_State *L, int dec_enc)
 	}
 }
 
+static int decrypt_encrypt_aead(lua_State *L, int dec_enc)
```

Comment:
I see this was the case already before: `dec_enc` is called `encdec` elsewhere, it's not obvious to me to begin with which one is 1 and which is 0 but this makes it worse. For a moment I had a mental rule "`1` is the one that is first in the variable name" but apparently it doesn't hold.

Maybe just having named constants like `#define ENCRYPTING 1` would help.